### PR TITLE
Add entry for Aruba Instant On

### PR DESCRIPTION
**Pull request by dhiltonp**
32x32 PNG generated from site favicon.

### CODE REVIEW

**Review by Carlgo11 — COMMENTED** [review]
Thanks for the pull request @dhiltonp!
There are a few things that need fixing:
1. Remove the subdomain from the “domain”-element 
2. An SVG is expected as the image format
3. IoT would be a better suited category/keyword.
4. As the documentation and recovery links are the same, please remove the recovery link.

Once that’s fixed a maintainer will approve and merge your PR. Best of luck and a happy new year to you!

**Comment by dhiltonp** [comment]
Thanks for your quick feedback.

I do think the subdomain is important, as there are different logins for different subdomains for the site. This subdomain is specifically for managing devices, while there is at least a separate community subdomain with independent credentials.

**Comment by dhiltonp** [comment]
I've switched to a SVG, I found the loading screen SVG and modified it.

With regards to PNG, `CONTRIBUTING.md` does say that PNGs are accepted, specifically: "If an SVG icon is not available, the icon should be a PNG with a resolution of...".

Looking at the test, it appears that PNGs are accepted, but must be explicitly linked in the json, or they may also be accepted as a secondary image type.

File: entries/p/portal.arubainstanton.com.json
Comment (hkamran80) [inline entries/p/portal.arubainstanton.com.json]:
```suggestion
  "Aruba Instant On": {
```

The name appears as "Aruba Instant On" for me

File: entries/p/portal.arubainstanton.com.json
Comment (hkamran80) [inline entries/p/portal.arubainstanton.com.json]:
```suggestion
    "domain": "arubainstanton.com",
```

Even though the community and the portal use separate authentication systems, the base domain should be listed.

File: img/p/portal.arubainstanton.com.svg
Comment (hkamran80) [inline img/p/portal.arubainstanton.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 21.9 22"><path fill="#ff8300" d="M11.009 0c1.06 0 1.92.86 1.92 1.92v5.99a1.92 1.92 0 0 1-3.84 0V1.92c0-1.06.86-1.92 1.92-1.92zm0 22h-1.4l-.6-.22-.43-.09-.57-.13-.48-.14-.32-.12h-.06a11 11 0 0 1-2.8-1.51 10.55 10.55 0 0 1-1.57-15.67 2.4 2.4 0 0 1 3.55 3.23 5.75 5.75 0 0 0 .88 8.57 6.1 6.1 0 0 0 1.59.86h.07l.14.05.28.08.29.08h.24l.38.05h.19c.393.03.787.03 1.18 0h.18l.38-.05h.23l.31-.08.26-.07.15-.06h.07a6.1 6.1 0 0 0 1.59-.86 5.79 5.79 0 0 0 2.4-4.68c.01-.2.01-.4 0-.6 0-.2-.05-.39-.09-.59a5.86 5.86 0 0 0-1.44-2.71 2.4 2.4 0 0 1 3.54-3.24 10.7 10.7 0 0 1 2.57 5c.074.353.127.71.16 1.07a10 10 0 0 1 .06 1.1 10.56 10.56 0 0 1-4.34 8.53 11 11 0 0 1-2.8 1.51h-.06l-.32.11-.48.14-.53.13-.42.09-.65.1h-.34c-.36.12-.7.12-.99.12z"/></svg>
```

Here's a minifed SVG

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: entries/p/portal.arubainstanton.com.json
Comment (dhiltonp) [reply entries/p/portal.arubainstanton.com.json]: So, the only reference to the subdomain should be in the name of this json file?

**Review by hkamran80 — CHANGES_REQUESTED** [review]
Please change the filename of the entry. It should be in the `a` folder with the filename `arubainstanton.com.json`

**Comment by dhiltonp** [comment]
How does that interact with other subdomains? 

If I do that, there is no information specifying that this is applicable to one subdomain and not another.

**Comment by hkamran80** [comment]
If the community was to be added, it would have the subdomain and a separate name.

**Review by hkamran80 — APPROVED** [review]

**Comment by dhiltonp** [comment]
Thanks!

**Pull request merged**
